modules/AppServiceEnvironments/2.0/deploy.json

```diff
@@ -0,0 +1,136 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+  "contentVersion": "2.0.0.0",
+  "parameters": {
+    "appServiceEnvironmentName": {
+      "type": "string",
+      "metadata": {
+        "description": "Required. The name of the App Service Environment."
+      }
+    },
+    "dnsSuffix": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Optional. The DNS Suffix to use for the default App Service Environment domain name."
+      }
+    },
+    "vnetResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Required. The Resource Group of the Virtual Network."
+      }
+    },
+    "vnetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Required. The Name of the Virtual Network to connect the App Service Environment."
+      }
+    },
+    "vnetSubnetName": {
```

Comment:
small change, please use subnetName as this is the variable name we've been using across the different VMs.

Thanks!

Reply:
Ok, I updated the parameter name to be 'subnetName' instead.